src/assets/css/themes/_variables.i386.scss

```diff
@@ -26,7 +26,7 @@ $danger: #aa0000;
 $info: #00aaaa;
 $warning: #aa00aa;
 $light: $gray-800;
-$dark: black;
+$dark: $gray-300;
```

Comment:
I hate making the dark colors so light, but it's how darkly gets around the issue. This is worth revisiting some point in the future.